server/public/plugin/hooks.go

```diff
@@ -382,4 +383,10 @@ type Hooks interface {
 	//
 	// Minimum server version: 9.5
 	OnSharedChannelsProfileImageSyncMsg(user *model.User, rc *model.RemoteCluster) error
+
+	// GenerateSupportData should be used by plugins which includes it's own content
+	// to the support packet.
+	//
+	// Minimum server version: 10.x
+	GenerateSupportData(c *Context) (*model.FileData, error)
```

Comment:
Where does the limit to only one file come from? Should we give plugins the flexibility to return a slice of files?

Reply:
You right that makes sense. I'll update accordingly.